.github/workflows/tests.yml

```diff
@@ -70,46 +76,65 @@ jobs:
           - python: "3.10"
             geos: 3.10.1
             speedups: 0
-          # dev
-          - python: "3.10"
-            geos: main
-            speedups: 1
```

Comment:
given that this is a maintenance branch, it does not need to test against the latest features.